### PR TITLE
Fix man page to show info on storage

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/podman.1.md [hunk 1]
```diff
@@ -42,11 +42,13 @@ When namespace is set, created containers and pods will join the given namespace
...
 **--root**=**value**
 
-Path to the root directory in which data, including images, is stored
+Storage root dir (default: "/var/lib/containers/storage" for UID 0, "$HOME/.local/share/containers/storage" for other users)
+Default root dir is configured in /etc/containers/storage.conf
 
 **--runroot**=**value**
 
-Path to the 'run directory' where all state information is stored
+Storage state dir (default: "/var/run/containers/storage" for UID 0, "/var/run/user/$UID/run" for other users)
```
Comment (mheon) [inline docs/podman.1.md]: Period at end of sentence

File: docs/podman.1.md [hunk 2]
```diff
@@ -42,11 +42,13 @@ When namespace is set, created containers and pods will join the given namespace
 
 **--root**=**value**
 
-Path to the root directory in which data, including images, is stored
+Storage root dir (default: "/var/lib/containers/storage" for UID 0, "$HOME/.local/share/containers/storage" for other users)
```
Comment (mheon) [inline docs/podman.1.md]: Period at end of sentence

File: docs/podman.1.md [hunk 3]
```diff
@@ -42,11 +42,13 @@ When namespace is set, created containers and pods will join the given namespace
 
 **--root**=**value**
 
-Path to the root directory in which data, including images, is stored
+Storage root dir (default: "/var/lib/containers/storage" for UID 0, "$HOME/.local/share/containers/storage" for other users).
```
Comment (TomSweeneyRedHat) [inline docs/podman.1.md]: Suggest:  "Storage root directory in which data, including images, is stored (default: ...)

File: docs/podman.1.md [hunk 4]
```diff
@@ -42,11 +42,13 @@ When namespace is set, created containers and pods will join the given namespace
...
+Default root dir is configured in /etc/containers/storage.conf.
 
 **--runroot**=**value**
 
-Path to the 'run directory' where all state information is stored
+Storage state dir (default: "/var/run/containers/storage" for UID 0, "/var/run/user/$UID/run" for other users).
```
Comment (TomSweeneyRedHat) [inline docs/podman.1.md]: Suggest: "Storage state directory where all state information is stored (default: ..."

File: docs/podman.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/podman.1.md]: Fixed

**Comment by baude** [comment]
rebase please

**Comment by umohnani8** [comment]
LGTM

**Comment by mheon** [comment]
Sure, LGTM

**Comment by rhatdan** [comment]
Has the tests actually run on this, or is this github acting up?
@baude or @mheon can you give me a `/lgtm`

**Pull request merged**